query/definable_resolution.rs

```diff
@@ -478,9 +457,7 @@ typedb_error! {
         RelatesNotFound(11, "The relation type '{label}' does not relate role '{role_name}'", label: Label, role_name: String),
         OwnsNotFound(12, "The type '{owner_label}' does not own attribute type '{attribute_label}'.", owner_label: Label, attribute_label: Label),
         PlaysNotFound(13, "The type '{player_label}' does not play the role '{role_label}'.", player_label: Label, role_label: Label),
-        ScopedValueTypeName(14, "Value type names cannot have scopes. Provided illegal name: '{scope}:{name}'.", scope: String, name: String),
-        ExpectedNonOptionalTypeSymbol(15, "Expected a type label or a type[] label, but not an optional type? label.\nSource:\n{declaration}", declaration: TypeRefAny),
-        ExpectedLabelButGotBuiltinValueType(16, "Expected type label got built-in value type name:\nSource:\n{declaration}", declaration: NamedType),
+        ExpectedNonVariableAndNonScopedTypeSymbol(15, "Expected a type label or a type[] label, but not a variable or scoped label.\nSource:\n{declaration}", declaration: TypeRefAny),
```

Comment:
Should we consolidate the numbering?

Reply:
Until 3.10 let's say we do

Reply:
We should start allocating gaps if we think we'll need them.